src/Components/WebAssembly/testassets/StandaloneApp/Pages/FetchData.razor

```diff
@@ -52,8 +52,13 @@ else
     protected override async Task OnParametersSetAsync()
     {
         startDate = StartDate.GetValueOrDefault(DateTime.Now);
-        forecasts = await Http.GetFromJsonAsync<WeatherForecast[]>(
-            $"sample-data/weather.json?date={startDate.ToString("yyyy-MM-dd")}");
+        var requestMessage = new HttpRequestMessage(HttpMethod.Get, $"sample-data/weather.json?date={startDate.ToString("yyyy-MM-dd")}");
+        requestMessage.SetBrowserRequestCache(BrowserRequestCache.NoCache);
```

Comment:
Please don't use `Components/WebAssembly/testassets/StandaloneApp` for testing or demonstrating random features :) It's meant to stay close to the default project template, as it's really a sandbox project not a test asset.

If you're adding this for an E2E test, could you do it inside `Components\test\testassets\BasicTestApp` instead?

Reply:
Also, there are a couple of redundant blank lines below.